uiTests/src/test/scala/com/virtuslab/gitmachete/uitest/UITestSuite.scala

```diff
@@ -21,14 +21,23 @@ object UITestSuite
     with GitMacheteExtension {
 
   override protected def baseFixture: IntelliJFixture = transformFixture {
-    val version = sys.props.get("intellij.version").filterNot(_.isEmpty).map(IntelliJVersion(_)).getOrElse(IntelliJVersion("2020.2.1"))
+    val version = sys.props.get("intellij.version").filterNot(_.isEmpty).map(IntelliJVersion(_, None)).getOrElse(IntelliJVersion("2020.2.1", None))
```

Comment:
y do we want to use such a default at this level? 
I assume that this is something that we will be required to bump sometimes (we get another place to watch), right?

Reply:
Yeah, maybe let's make this `intellij.version` param required as for now (and patch `run-ui-tests` accordingly)

Reply:
i've done that. Let me know if you want to handle it in such a way.